cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -280,6 +280,7 @@ func createCoreDNSAddon(deploymentBytes, serviceBytes, configBytes []byte, clien
 	canMigrateCorefile := true
 
 	if corefileMigrationRequired {
+		//If migration is required, try and migrate the Corefile
```

Comment:
We get in here for default corefiles that would require migration (e.g. when new defaults are added, or default options removed)